report_compassion/models/contract.py

```diff
@@ -0,0 +1,102 @@
+##############################################################################
+#
+#    Copyright (C) 2016 Compassion CH (http://www.compassion.ch)
+#    Releasing children from poverty in Jesus' name
+#    @author: Emanuel Cino <ecino@compassion.ch>
+#
+#    The licence is in the file __manifest__.py
+#
+##############################################################################
+from odoo import api, models, fields
+from datetime import date
+
+
+class AccountInvoice(models.Model):
+    """
+    Make Invoice translatable for communications with dates.
+    """
+
+    _inherit = ["account.move", "translatable.model"]
+    _name = "account.move"
+
+    # Gender field is mandatory for translatable models
+    gender = fields.Char(compute="_compute_gender")
+
+    def _compute_gender(self):
+        for i in self:
+            i.gender = "M"
```

Comment:
I understand that it's currently the way to make it translatable, but it is ugly to do so in my opinion, isn't there a better way? I mean, it's weird to have a gender field for an AccountInvoice class